frontend/apps/reader/modules/readercoptlistener.lua

```diff
@@ -134,7 +134,7 @@ function ReaderCoptListener:onResume()
         return
     end
 
-    ReaderCoptListener:rescheduleHeaderRefreshIfNeeded()
+    self:headerRefresh()
```

Comment:
Aren't you circumventing/sabotaging the very reason that function exists this way?

Reply:
Needs to apply below to OutOfScreenSaver, too.

(c.f., https://github.com/koreader/koreader/pull/7624/commits/08b0e2b1d510f4a654b369c36e1848a4460b24c8)

----

@Frenzie: headerRefresh calls rechedule, too, and checks if the header actually benefits from a refresh *now* first. This is similar in spirit to what the Footer does.

Reply:
Alright, good. :+1:

Reply:
> Needs to apply below to OutOfScreenSaver, too.
> 
> (c.f., [08b0e2b](https://github.com/koreader/koreader/commit/08b0e2b1d510f4a654b369c36e1848a4460b24c8))

I have added it on `OutOfScreenSaver`, but cannot test this.

Reply:
No worries, the two are mutually exclusive, and Footer does the same thing ;).